tests/test_iocdb_search.py

```diff
@@ -0,0 +1,18 @@
+import pytest
+
+
+@pytest.fixture
+def mock_list():
```

Comment:
I'd recommending using a more explicit name than `mock_list`. What is it a list of?